src/revlanguage/distributions/phylogenetics/tree/Dist_FBDP.cpp

```diff
@@ -97,7 +97,7 @@ RevBayesCore::AbstractBirthDeathProcess* Dist_FBDP::createDistribution( void ) c
     // death rate
     RevBayesCore::DagNode* d_s = mu->getRevObject().getDagNode();
     // serial sampling rate
-    RevBayesCore::DagNode* s_s = psi->getRevObject().getDagNode();
+    RevBayesCore::DagNode* s_s = phi->getRevObject().getDagNode();
```

Comment:
I'm perplexed by the psi -> phi renaming.

Reply:
This was to avoid double usage of the variable psi. There were situations were internally a new local variable psi was used for the tree, e.g., when simulating trees. I think that it might be dangerous to use local variables with the same name, so we changed it.